packages/auth-construct/src/construct.ts

```diff
@@ -980,19 +980,6 @@ export class AmplifyAuth
         output.mfaTypes = JSON.stringify(mfaTypes);
       }
     }
-    const oAuthMappings = this.providerSetupResult.oAuthMappings;
-    if (oAuthMappings[authProvidersList.amazon]) {
-      output.amazonClientId = oAuthMappings[authProvidersList.amazon];
-    }
-    if (oAuthMappings[authProvidersList.facebook]) {
-      output.facebookClientId = oAuthMappings[authProvidersList.facebook];
-    }
-    if (oAuthMappings[authProvidersList.google]) {
-      output.googleClientId = oAuthMappings[authProvidersList.google];
-    }
-    if (oAuthMappings[authProvidersList.apple]) {
-      output.appleClientId = oAuthMappings[authProvidersList.apple];
-    }
```

Comment:
I don't think we should do this.

If somebody updates and deploys this will result in deletion of cfn.Outputs which might be seen as data loss should somebody consume them out of bound. This would be runtime breaking change in addition to schemas compile time break.

I think we should mark these fields as deprecated and start a list of changes we want to ship at next MV.

Reply:
We don't claim/document that the raw stack outputs should be used directly. Customers _could_ use them directly, but they could also read any other part of the stack that is "internal" to our synth process. As long as customers are using our defined interfaces in `client-config` or `deployed-backend-client`, they won't be broken by this change.

Reply:
Because it's not really a public contract that we expose (e.g. customers relying on the `/lib/...` artifacts from our packages that we can change anytime) and given that the new schema can still read stack outputs that have the `clientId` set, I'm fine going ahead with this change.